src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -8,14 +8,21 @@
 @import '../../styles/shared-styles.scss';
 @import '../../../node_modules/office-ui-fabric-core/dist/sass/References';
 
-$font-size: var(--font-size, #{$ms-font-size-m});
-$font-weight: var(--font-weight, #{$ms-font-weight-semibold});
-$color: var(--color, #{$ms-color-neutralDark});
+$person-card-display-name-font-size: var(--person-card-display-name-font-size, 27.3103px);
+$person-card-secondary-title-font-size: var(--person-card-secondary-title-font-size, 18.2069px);
+$person-card-additional-details-title-font-size: var(--person-card-additional-details-title-font-size, 14px);
+$person-card-additional-details-item-font-size: var(--person-card-additional-details-item-font-size, 15px);
+$person-card-display-name-color: var(--person-card-display-name-color, #333333);
+$person-card-secondary-title-color: var(--person-card-secondary-title-color, #767676);
+$person-card-additional-details-title-color: var(--person-card-additional-details-title-color, #000000);
+$person-card-additional-details-item-color: var(--person-card-additional-details-item-color, #201f1e);
+$person-card-background-color: var(--person-card-background-color, #ffffff);
 
 :host .root {
   position: relative;
+  background: $person-card-background-color;
+  box-shadow: 0px 0px 4.55172px rgba(0, 0, 0, 0.16);
```

Comment:
Box shadow should be removed since the flyout handles the shadow now

Reply:
Removed added back in the width modifier from flyout:
`width: var(--mgt-flyout-set-width, 340px)`

Something to consider for the documentation is adding a link or short section on components that utilize mgt-flyout